src/main/java/org/jitsi/jicofo/codec/JingleOfferFactory.java

```diff
@@ -434,51 +434,75 @@ private static void addAudioToContent(
+
+            int opusMaxAverageBitrate = jitsiMeetConfig.getOpusMaxAverageBitrate();
+            if (opusMaxAverageBitrate != -1)
+            {
+                addParameterExtension(
+                        opus,
+                        "maxaveragebitrate",
+                        String.valueOf(opusMaxAverageBitrate));
+            }
+
+            if (config.opus.useInbandFec())
+            {
+                // fmtp:111 useinbandfec=1
+                addParameterExtension(opus, "useinbandfec", "1");
+            }
+
+            if (config.tcc.enabled() && jitsiMeetConfig.isTccEnabled())
+            {
+                // a=extmap:5 http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01
+                RTPHdrExtPacketExtension tcc = new RTPHdrExtPacketExtension();
+                tcc.setID(String.valueOf(config.tcc.id()));
+                tcc.setURI(URI.create(RTPExtension.TRANSPORT_CC_URN));
+                rtpDesc.addExtmap(tcc);
+
+                // a=rtcp-fb:111 transport-cc
+                opus.addRtcpFeedbackType(createRtcpFbPacketExtension("transport-cc", null));
+            }
         }
 
-        int opusMaxAverageBitrate = jitsiMeetConfig.getOpusMaxAverageBitrate();
-        if (opusMaxAverageBitrate != -1)
+        if (config.icas16.enabled())
         {
-            addParameterExtension(
-                    opus,
-                    "maxaveragebitrate",
-                    String.valueOf(opusMaxAverageBitrate));
+            // a=rtpmap:103 ISAC/16000
+            addPayloadTypeExtension(rtpDesc, config.icas16.pt(), "ISAC", 16000);
         }
 
-        // fmtp:111 useinbandfec=1
-        addParameterExtension(opus, "useinbandfec", "1");
-
-        if (config.tcc.enabled() && jitsiMeetConfig.isTccEnabled())
+        if (config.icas32.enabled())
         {
-            // a=extmap:5 http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01
-            RTPHdrExtPacketExtension tcc = new RTPHdrExtPacketExtension();
-            tcc.setID(String.valueOf(config.tcc.id()));
-            tcc.setURI(URI.create(RTPExtension.TRANSPORT_CC_URN));
-            rtpDesc.addExtmap(tcc);
-
-            // a=rtcp-fb:111 transport-cc
-            opus.addRtcpFeedbackType(createRtcpFbPacketExtension("transport-cc", null));
+            // a=rtpmap:104 ISAC/32000
+            addPayloadTypeExtension(rtpDesc, config.icas32.pt(), "ISAC", 32000);
         }
 
-        // a=rtpmap:103 ISAC/16000
-        addPayloadTypeExtension(rtpDesc, 103, "ISAC", 16000);
-
-        // a=rtpmap:104 ISAC/32000
-        addPayloadTypeExtension(rtpDesc, 104, "ISAC", 32000);
-
-        // rtpmap:126 telephone-event/8000
-        addPayloadTypeExtension(rtpDesc, 126, Constants.TELEPHONE_EVENT, 8000);
+        if (config.telephoneEvent.enabled())
+        {
+            // rtpmap:126 telephone-event/8000
+            addPayloadTypeExtension(rtpDesc, config.telephoneEvent.pt(), Constants.TELEPHONE_EVENT, 8000);
+        }
```

Comment:
Any desire to add support for any other audio codecs?  Or would that be a separate PR?

Reply:
That would be separate